homeassistant/components/roon/media_player.py

```diff
@@ -164,6 +152,13 @@ def supported_features(self):
         """Flag media player features that are supported."""
         return SUPPORT_ROON
 
+    @property
+    def group_members(self):
+        """Return the grouped players."""
+
+        roon_names = self._server.roonapi.grouped_zone_names(self._output_id)
```

Comment:
Is the order set with the leader first here?

Reply:
It is.